packages/atomic/src/components/search/atomic-rating-facet/atomic-rating-facet.spec.ts

```diff
@@ -0,0 +1,553 @@
+        props: {
+          field: 'testfield',
+          numberOfIntervals: 7,
+          filterFacetCount: false,
+          injectionDepth: 500,
+        },
+      });
+
+      expect(buildNumericFacet).toHaveBeenCalledWith(element.bindings.engine, {
+        options: expect.objectContaining({
+          field: 'testfield',
+          numberOfValues: 7,
+          filterFacetCount: true,
+          injectionDepth: 500,
+          generateAutomaticRanges: false,
+          sortCriteria: 'descending',
+        }),
+      });
+    });
+
+    it('should call buildSearchStatus with engine', async () => {
+      const {element} = await renderRatingFacet();
+      expect(buildSearchStatus).toHaveBeenCalledWith(element.bindings.engine);
+    });
+
+    it('should call buildTabManager with engine', async () => {
+      const {element} = await renderRatingFacet();
+      expect(buildTabManager).toHaveBeenCalledWith(element.bindings.engine);
+    });
+
+    it('should register facet in store', async () => {
+      await renderRatingFacet();
+      expect(mockedRegisterFacet).toHaveBeenCalled();
+    });
+  });
+
+  describe('props validation', () => {
+    // TODO V4: KIT-5197 - Remove skip
+    it.skip.each<{
+      prop:
+        | 'displayValuesAs'
+        | 'numberOfIntervals'
+        | 'minValueInIndex'
+        | 'headingLevel'
+        | 'injectionDepth';
+      invalidValue: string | number;
+    }>([
+      {
+        prop: 'displayValuesAs',
+        invalidValue: 'invalid',
+      },
+      {
+        prop: 'numberOfIntervals',
+        invalidValue: 0,
+      },
+      {
+        prop: 'minValueInIndex',
+        invalidValue: -1,
+      },
+      {
+        prop: 'headingLevel',
+        invalidValue: 7,
+      },
+      {
+        prop: 'injectionDepth',
+        invalidValue: -1,
+      },
+    ])(
+      'should set error when #$prop is invalid',
+      async ({prop, invalidValue}) => {
+        const element = await renderRatingFacet().then(({element}) => element);
+
+        expect(element.error).toBeUndefined();
+
+        // biome-ignore lint/suspicious/noExplicitAny: testing invalid values
+        (element as any)[prop] = invalidValue;
+        await element.updateComplete;
+
+        expect(element.error).toBeDefined();
+        expect(element.error.message).toMatch(new RegExp(prop, 'i'));
+      }
+    );
+
+    it('should warn when both tabsIncluded and tabsExcluded are provided', async () => {
+      const consoleWarnSpy = vi
+        .spyOn(console, 'warn')
+        .mockImplementation(() => {});
+
+      await renderRatingFacet({
+        props: {
+          tabsIncluded: ['tab1'],
+          tabsExcluded: ['tab2'],
+        },
+      });
+
+      expect(consoleWarnSpy).toHaveBeenCalledWith(
+        expect.stringContaining('tabs-included')
+      );
+      consoleWarnSpy.mockRestore();
+    });
+
+    it('should handle maxValueInIndex defaulting to numberOfIntervals', async () => {
```

Comment:
this test pretty much duplicates 'should default maxValueInIndex to numberOfIntervals'